app/routes/web.php

```diff
@@ -5,5 +5,42 @@
 Route::get('terms-of-use', function() { return View::make('pages.terms_of_use'); });
 Route::get('privacy-policy', function() { return View::make('pages.privacy_policy'); });
 Route::get('faq', 'FaqController@index');
```

Comment:
We can tackle this problem with routes/web.php after you get those "readme file change" commits removed.  I'll give tips regarding this after you do that.

You just added the following 2 lines to web.php, right?
```
Route::get('edit-my-location/{location_id}', 'LocationManagementController@editLocation');
Route::post('edit-location', 'LocationManagementController@editLocationSave');
```

Reply:
yes